### ISSUE TITLE
Document columns for _cat/indices and _cat/shards

### ISSUE THREAD

**Issue by joegallo** [elastic/elasticsearch#98955]
Related to #95092 / #98622

The docs for `_cat/shards` ([link](https://www.elastic.co/guide/en/elasticsearch/reference/current/cat-shards.html#cat-shards-query-params)) mention **some** of the various columns that are available for the `h` parameter, but the docs for `_cat/indices` do not ([link](https://www.elastic.co/guide/en/elasticsearch/reference/current/cat-indices.html#cat-shards-query-params)).

Taking a look at the code for those actions, though ([`RestShardsAction.java`](https://github.com/dakrone/elasticsearch/blob/dc4debc5784c709db23180e103d8e2bf0476a6d6/server/src/main/java/org/elasticsearch/rest/action/cat/RestShardsAction.java) and [`RestIndicesAction.java`](https://github.com/dakrone/elasticsearch/blob/dc4debc5784c709db23180e103d8e2bf0476a6d6/server/src/main/java/org/elasticsearch/rest/action/cat/RestIndicesAction.java)), it turns out that `_cat/shards` supports more than 150 columns, of which ~60 are documented, while `_cat/indices` supports nearly 300.

We may want to more systematically document the available columns that these and other `_cat` APIs support.

**Comment by georgewallace** [comment]
related to https://www.notion.so/bumpsh/Doc-enum-values-missing-1ea1fc8d70178041a82fe9ad4c7bfd1c?pvs=4

**Comment by kosabogi** [comment]
> [@kosabogi](https://github.com/kosabogi) and [@charlotte-hoblik](https://github.com/charlotte-hoblik) this work is completed already?

Yes, we had a separate issue to track this: https://github.com/elastic/elasticsearch-specification/issues/4345

**Comment by georgewallace** [comment]
All of these types have been corrected and added to the docs.
